sim/core/focus.go

```diff
@@ -73,85 +184,91 @@ func (fb *focusBar) SpendFocus(sim *Simulation, amount float64, metrics *Resourc
 	}
 
 	newFocus := fb.currentFocus - amount
-	metrics.AddEvent(-amount, -amount)
+	//metrics.AddEvent(-amount, -amount)
 
 	if sim.Log != nil {
-		fb.unit.Log(sim, "Spent %0.3f focus from %s (%0.3f --> %0.3f).", amount, metrics.ActionID, fb.currentFocus, newFocus)
+		//fb.unit.Log(sim, "Spent %0.3f focus from %s (%0.3f --> %0.3f).", amount, metrics.ActionID, fb.currentFocus, newFocus)
```

Comment:
i think commenting this will remove focus from the timeline as the timeline is parsed from logs

Reply:
which makes me think you probably want to look into the TS files to see where we parse resource metrics if you have to add something new for focus (even tho focus is not a new resource so should be there)

Reply:
No the focus timeline works, idk why that one is needed but i can add it back np. Had some lag issues with the resource metrics before I think thats why I commented them out

Reply:
I dont have them now though. Thats the issue with old and big branches :D Things are not the same as when I started. I'll just add them back in